torch/onnx/symbolic_opset9.py

```diff
@@ -307,29 +309,43 @@ def reduce_dim(g, self, dim, keepdim, dtype):
         return reduce_nodim, reduce_dim
     return reduce
 
+
 sum = _reduce_with_dtype('ReduceSum', 'sum')
 mean = _reduce_with_dtype('ReduceMean', 'mean')
 prod = _reduce_with_dtype('ReduceProd', 'prod', allow_multi_dim_support=False)  # torch.prod does not support multidimensional 'dim'
 
+
 @parse_args('v', 'i', 'none')
 def cumsum(g, input, dim, dtype):
-    if dtype.node().kind() != 'prim::Constant':
-        return _unimplemented(name, "dtype")
-    return g.op("ATen", input, operator_s="cumsum", dim_i=dim)
+    if sym_help._operator_export_type == torch.onnx.OperatorExportTypes.ONNX_ATEN_FALLBACK:
+        if dtype.node().kind() != 'prim::Constant':
+            return _unimplemented(name, "dtype")
+        return g.op("ATen", input, operator_s="cumsum", dim_i=dim)
+    else:
+        raise RuntimeError('Unsupported: ONNX export of cumsum in '
```

Comment:
maybe we should abstract out this error message, like `_opset_unsupported('cumsum', current_opset=9, supported_opset=11)`